code/game/atoms.dm

```diff
@@ -68,6 +68,10 @@
 	/// Radiation insulation types
 	var/rad_insulation = RAD_NO_INSULATION
 
+  ///The material this atom is made of, used by a lot of things like furniture, walls, and floors (if I finish the functionality, that is.)
+	var/datum/material/material 

```

Comment:
For the sake of avoiding future pain of multiple materials make this a list for now. It can just be a single entry list